src/components/App/App.vue

```diff
@@ -21,13 +25,27 @@ export default {
     Container
   },
   data: () => ({
-    locations: data.locationData,
-    cities: data.cityData.data,
-    states: data.stateData.data
+    locations: [],
+    cities: [],
+    states: []
   }),
+  mounted: function () {
+    this.$nextTick(function () {
+      fetchAPI.getStates().then(data => {
+        this.states = data.data
+      })
+    })
+  },
   methods: {
-    debug (input) {
-      console.log(input)
+    retrieveCities (state) {
+      fetchAPI.getCities(state).then(data => {
+        this.cities = data.data
+      })
+    },
+    retrieveLocale (location) {
+      fetchAPI.getLocale(location.city, location.state).then(data => {
+        this.locations.push(data.data)
```

Comment:
Agreed, will definitely need to assign ids for the "delete a card" user story.

Reply:
I do think I'm following and on board with this idea - we'd definitely want that unique identifier to delete by.
 just checking,
so are we thinking each locale is created as an obj literal with properties of `city` and `state`, as well as an `id` property?
Also makes me wonder - does this seem like an argument for having saved _locations_  rather than static individual cards persisting (maybe separated to a different view, or updating the data for that card on a new page load). Maybe it doesn't change depending on time between a user's checks and the API update, but it seems like that could be more useful than having cards saved which hold explicitly outdated information to display. Additionally, would there ever be a case in which we would want to be able to show two cards for the same location but with different timestamps?